rpmlint/checks/ZyppSyntaxCheck.py

```diff
@@ -0,0 +1,18 @@
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class ZyppSyntaxCheck(AbstractCheck):
+    def check(self, pkg):
+        # We care only about the names, versions are pointless here
+        pkg_supplements = list(map(lambda x: x[0], pkg.supplements))
```

Comment:
I dont think map and lambda is needed `pkg_supplements = [x for x in pkg.supplements]` and for others same